### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by AdedayoOpeyemi**
This pull request is to merge the Testing branch with the development branch

Requirments

- Set up React Testing Library.
- Write unit tests for the files operate.js and calculate.js using Jest.
- Create unit tests for all React components:
- Use Jest snapshots to test the components.
- Use React Testing Library to simulate user interaction.
- Run your full tests suite using Jest. All tests should pass.

### CODE REVIEW

File: src/logic/test/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import operate from '../operate';
+
```
Comment (eliasecasta) [inline src/logic/test/operate.test.js:2]: - [ ] You are missing to test the division by 0 output as well as the % mod operation, please do so to comply with the project requirements.

File: src/logic/test/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import calculate from '../calculate';
+
```
Comment (eliasecasta) [inline src/logic/test/calculate.test.js:2]: - [ ] Please cover the remaining functionalities of this script like the +/- and the dot functionality to comply with the project requirements.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @AdedayoOpeyemi and @JbirdL86, I am Elias, your reviewer for this PR

Well done so far with your project!

The snapshot testing is done, you are only missing to extend the logic tests.
 
I will mention to you a small summary of things that need to be changed in order to keep moving:

- [ ] Please extend your logic tests to comply with the project requirements

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by EricMbouwe — APPROVED** [review]
Hi @AdedayoOpeyemi, 

# Status: Approved✔️

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**